content/docs/100-get-started/200-build-an-app.mdx

```diff
@@ -3,7 +3,7 @@ title: Build an App
 excerpt: A quick tutorial that covers the basics of building your first app.
 ---
 
-This tutorial will walk you through the basics of building a Raydiant app. We’ll use the [Simple Weather](https://simple-weather-raydiant-app.vercel.app/?location=New%20York) app to set up its inputs and preview the app in the Raydiant Dashboard.
+This tutorial will walk you through the basics of building a Raydiant app. We’ll use the [Weather](https://simple-weather-raydiant-app.vercel.app/?location=New%20York) app to set up its inputs and preview the app in the Raydiant Dashboard.
```

Comment:
I was going call the new sample app "Simple Video" but instead just decided to drop "Simple" altogether. I don't think adds value and makes the sample apps easier to scan.